common/src/main/java/org/apache/atlas/repository/Constants.java

```diff
@@ -75,6 +75,7 @@ public final class Constants {
     public static final String TYPEVERSION_PROPERTY_KEY     = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.version");
     public static final String TYPEOPTIONS_PROPERTY_KEY     = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.options");
     public static final String TYPESERVICETYPE_PROPERTY_KEY = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.servicetype");
+    public static final String TYPE_SKIP_DISPLAYNAME_UNIQUENESS_CHECK = getEncodedTypePropertyKey(INTERNAL_PROPERTY_KEY_PREFIX + "type.skipDisplayNameUniquenessCheck");
```

Comment:
skipDisplayNameUniquenessCheck -> allowDuplicateDisplayName

I suggest changing this flag to allowDuplicateDisplayName, this will also need to update the logic a bit